support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -175,17 +182,26 @@ const ContributionThankYou = ({
   countryId,
   campaignCode,
   thankyouPageHeadingTestVariant,
+  largeDonationMessageTestVariant,
 }: ContributionThankYouProps) => {
   const isKnownEmail = guestAccountCreationToken === null;
-  const campaignSettings = useMemo<CampaignSettings | null>(() => getCampaignSettings(campaignCode));
+  const campaignSettings = useMemo<CampaignSettings | null>(() =>
+    getCampaignSettings(campaignCode));
   const isUsEndOfYearAppeal = countryId === 'US';
 
+  const isLargeUSDonation =
+    countryId === 'US' && isLargeDonation(amount, contributionType);
+
+  const shouldShowLargeDonationMessage =
+    largeDonationMessageTestVariant && isLargeUSDonation;
+
   useEffect(() => {
     trackUserData(
       paymentMethod,
       contributionType,
       user.isSignedIn,
       isKnownEmail,
+      isLargeUSDonation,
```

Comment:
we're doing this to ease analysis later on. Ideally we would exclude non large donators from the ab test, but we can't :(